.env

```diff
@@ -6,6 +6,7 @@ REACT_APP_API_BASE_URL=http://localhost:5000/api
 REACT_APP_LOGIN_URL=http://localhost:5000/api/login
 REACT_APP_LOGOUT_URL=http://localhost:5000/api/logout
 REACT_APP_ERFPACHT_URL=https://example.org/erfpachtdingen
+REACT_APP_ATLAS_API_BASE_URL=http://localhost:5000/api
```

Comment:
Why is this on localhost? This should always be api.data.amsterdam, I think.

Reply:
Nope, I've mocked the atlas search in the mock api.